### PR TITLE
Call out ExO cmdlets when being used

### PR DESCRIPTION
**Pull request by AndySDay**
This is a long article and a customer has asked for us to add the need to use Exchange Online Powershell alongside SCC Powershell in the examples where ExO cmdlets are being used. The general need for ExO PS is written at the top of the article, but it would help to call-out where ExO cmdlets are being used in the examples, e.g. examples that call $DG = Get-DistributionGroup. The confusion has been that SCC also contains/contained the same PS cmdlets as ExO, but that those did not work from the SCC side.

### CODE REVIEW

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/compliance/permissions-filtering-for-content-search.md [hunk 1]
```diff
@@ -150,7 +150,7 @@ This example allows members of the "US Discovery Managers" role group to perform
 New-ComplianceSecurityFilter -FilterName USDiscoveryManagers  -Users "US Discovery Managers" -Filters "Mailbox_CountryCode  -eq '840'" -Action All
 ```
 
-This example allows members of the eDiscovery Manager role group to search only the mailboxes of members of the Ottawa Users distribution group. 
+This example allows members of the eDiscovery Manager role group to search only the mailboxes of members of the Ottawa Users distribution group. Get-DistributionGroup in Exchange Online Powershell is used to find the members of Ottawa Users.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/permissions-filtering-for-content-search.md]:
```suggestion
This example allows members of the eDiscovery Manager role group to search only the mailboxes of members of the Ottawa Users distribution group. Get-DistributionGroup in Exchange Online PowerShell is used to find the members of Ottawa Users.
```

File: microsoft-365/compliance/permissions-filtering-for-content-search.md [hunk 2]
```diff
@@ -160,7 +160,7 @@ $DG = Get-DistributionGroup "Ottawa Users"
 New-ComplianceSecurityFilter -FilterName DGFilter  -Users eDiscoveryManager -Filters "Mailbox_MemberOfGroup -eq '$($DG.DistinguishedName)'" -Action Search
 ```
 
-This example prevents any user from deleting content from the mailboxes of members of the Executive Team distribution group.
+This example prevents any user from deleting content from the mailboxes of members of the Executive Team distribution group. Get-DistributionGroup in Exchange Online Powershell is used to find the members of Executive Team.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/permissions-filtering-for-content-search.md]:
```suggestion
This example prevents any user from deleting content from the mailboxes of members of the Executive Team distribution group. Get-DistributionGroup in Exchange Online PowerShell is used to find the members of the Executive Team.
```

Comment (JohanFreelancer9) [inline microsoft-365/compliance/permissions-filtering-for-content-search.md]:
```suggestion
This example prevents any user from deleting content from the mailboxes of members of the Executive Team distribution group. Get-DistributionGroup in Exchange Online PowerShell is used to find the members of the Executive Team.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by markjjo** [comment]
@AndySDay PR merged. Thanks for submitting. 

@JohanFreelancer9  Thanks for editing.

**Pull request merged**